src/components/DetailsView/DetailsView-view.tsx

```diff
@@ -13,12 +13,22 @@ import View from '../View';
 const DetailsView = (props: DetailsViewProps) => {
     const classes = useDetailsMenuStyles();
     const address = props.match.params.walletHash || '';
+    const wallet = props.match.params.walletSource;
+    const limit = props.match.params.limit;
+    const groupBy = props.match.params.groupBy;
```

Comment:
Może destrukturyzacja zamiast kilka razy "props.match.params"?

Reply:
aaa to moj pr :)